Android: Update all translations for 0.5

carstene1ns opened the pull request:
- This adds `Belarusian`, `Ukrainian` and `Russian`
- Ignore the whitespace change in `Thai`, it was simply forgotten to change before...
- One string is still missing in `Chinese (Taiwan)`, hope this gets added before the deadline (will rebase this then)
- GitHub does not render the `Korean` symbols for me here. I can assure they are there in the file/repository.

Note, that I removed some `how_to_use_easy_rpg_explanation` strings... These have been outdated for roughly 8 months. I added an issue on transifex shortly after the app could read from multiple folders and would recurse in subfolders, but these were never changed over half a year later. I think having a complete and correct english text is better than having a broken and outdated translation in the main "FAQ".
This only affects the incomplete (<80%) translations on transifex of course (their maintainers gone M.I.A.).

BlisterB commented:
Hi guys, I've done french translation :).

Ghabry commented:
@rohkea 
Don't worry, there are still one or two days left for changes.

@BlisterB 
Thanks :)

rohkea commented:
N.B. Please note that zh_TW is **not** Taiwanese, it's Standard Mandarin as used on Taiwan. It's the standard language based on the Beijing dialect, popularised in Taiwan in mid-20th century. 'Taiwanese' usually refers to Taiwanese Hokkien, a variety of Min Nan spoken in Taiwan. Most software doesn't feature Taiwanese translations. Taiwanese would probably use a 'nan' or 'nan_TW' ISO code if someone adds such a translation.

Saying that EasyRPG has a Taiwanese translation is misleading.

carstene1ns commented:
@rohkea: Well, yeah, was an error on my side ("brain fart" xD).
Transifex does it right and just names it `Chinese (Taiwan)`. I had used the ISO codes before in the pull request description and changed them shortly before submitting... When first adding it, I did it right (#862).
Anyway, changed it now, but i guess nobody would have noticed that anyway (because this description will show up nowhere besides here).

carstene1ns commented:
(will rebase the last 2 commits into the first 2 later)

carstene1ns commented on builds/android/app/src/main/res/values-fr/strings.xml:
```diff
@@ -24,36 +23,53 @@
+Vous pouvez définir de nouveaux dossiers de jeux via les paramètres.
+
+Optionnel :\n
+Placez les RTP dans les dossier /easyrpg/rtp/2000 et /easyrpg/rtp/2003.</string>
```
@BlisterB: Seems you forgot half of the translation here. :-1: 
No word about subfolders, EXE, ZIP,etc...

BlisterB replied:
That was intentional : the half of the original text is now useless. There is no problem anymore with subfolders because Ghabry implemented a system scanning subfolders.

But you're right I forgot the EXE and ZIP part. I've fixed that.

Ghabry replied:
Please respect the translation note: "Please take care of \n (linebreak) and \t (tabulator). Don't add manual linebreaks or whitespace after \n. Don't translate: /easyrpg/games/, /easyrpg/rtp/2000 and /2003"

Ghabry replied:
If you think this is stupid and looks ugly: Thank Android, when you linebreak manually you get ugly trailing whitespace on a new line :(

BlisterB replied:
Arf I thought Transifex would do this for me (it displays an "enter" icon in preview), apologies. Fixed.

For the translation of the path, it seems that you're reffering to the old translation. Besides, the path is not /easyrpg/game but, well, android.

BlisterB replied:
Be carefull there is some lines on the base .xml that doesn't respect the rules you said, about the manual linebreak : 

"Thank you for supporting EasyRPG Player.\n⏎
A logfile and your savegames were attached to this E-Mail.\n⏎
Please don't remove the logfile! We need a savegame that is close to the bug location.\n⏎
Please tell us in detail what went wrong.\n\n⏎"

"No RPG Maker 2000/2003 games found.⏎
You can add additional game directories in the settings."

Ghabry replied:
These two cases don't expose the problem because they are not in a message box.

1st is mail
2nd is error case in game browser

Ghabry replied:
And the path is /easyrpg/games on the Android device, what do you mean with path is "android"?

BlisterB replied:
The path is not android, I was saying "but well, android" like we sometimes says "android stupid stuff" :p.
The path is different depending of API and their storage politics.

For exemple, my default path is /storage/emulated/0/easyrpg/games. (Android 6.0).

carstene1ns replied:
Yeah, in the text it just means the subfolder from the main storage folder:
main: `/storage/emulated/0/`
sub: `easyrpg/games`

Ghabry replied:
I talked about from the perspective of a PC user who transfers his games to Android.
He will see easyrpg/games when connecting it to the PC. Because this is the most sane workflow in getting games on Android ;)

BlisterB replied:
I understood :p.
But hey android, give me another headache : with a lot of smartphone with 6.0 when you connect them to a PC, you cannot reach anything other than the SD card. (This is actually not a joke).

To be perfect, we should just put a thing like %s in the text, to replace it with the proper path programmatically. To avoid any problem, like people using the custom DirChooser looking for /easyrpg/games, or people using a PC and pasting the game folder in the wrong place.
But this is not really important, just to avoid some dumb review on Google Playstore.

Ghabry replied:
This will not help, people usually find there games by now but then they ask how to run Ib.rar ;)

BlisterB commented:
Why taunting in commit description -___-.

Ghabry commented:
He will merge all the commits in the end to one single, don't worry :P

BlisterB commented:
Yeah I assumed he'll do that, I just don't understand the need to taunt.
http://www.urbandictionary.com/define.php?term=FUR

Ghabry commented:
I'm quite sure this was a play on words on FFFFFUUUUUUUUUUUUU Meme >.>

carstene1ns commented:
wat.
![rage](https://picload.org/image/rdgdpwoa/rage.png)

french → fr
rage → FFFFFUUUUUU

BlisterB commented:
Apologies, F.U.R was the first pop culture references I had in mind.

carstene1ns commented:
Okay, no changes for a week since then, final rebase time and then can be merged.

Ghabry commented on builds/android/app/src/main/res/values-de/strings.xml:
```diff
@@ -35,32 +35,25 @@
-\n
-Spezielle Funktionen:\n
-Im Einstellungsmenü kannst du Tasten verschieben und neue hinzufügen. Drücke im Layout-Editor die Zurück-Taste deines Gerätes, um ein Menü zu öffnen.</string>
+\t/easyrpg/games/MeinSpiel.zip ist FALSCH. ZIP und EXE werden nicht unterstützt.</string>
```
Hhng, just saw these two incorrect linebreaks here :(

Ghabry commented:
About the linebreaks in the mail text: One mail program adds an extra whitespace because of the \n+return:
Yahoo Mail hhhhhhhng

carstene1ns commented:
Good to know it is not our fault :^)
